### ISSUE TITLE
Re#736 - Form Elements, Basic (Pt 1)

### ISSUE THREAD

**Issue by wesruv** [backdrop/backdrop-issues#753]
Implement the following form element styles as per the style guide:
- Text field
- Text field (disabled)
- Text field (autocomplete)
- Text field (machine name)
- Text area
- Date
- File 
- File (managed)
- Password
- Password confirm

See master issue for instructions/guidelines for implementation: 
https://github.com/backdrop/backdrop-issues/issues/736

![forms-basic](https://cloud.githubusercontent.com/assets/397895/7218344/bac38bb0-e61e-11e4-88a7-0339f37ab17a.png)

**Comment by jenlampton** [comment]
I updated the style guide to unblock this issue.

**Comment by Graham-72** [comment]
I have added a commit to PR https://github.com/backdrop/backdrop/pull/819 in order to modify relevant styles in core/themes/seven/css/styles.css

**Comment by wesruv** [comment]
Assigning to @Graham-72 to prevent confusion

**Comment by Graham-72** [comment]
I have produced a new PR backdrop/backdrop/pull/830
There are other changes to come for style.css as I review other issues
e.g. https://github.com/backdrop/backdrop-issues/issues/738

**Comment by jenlampton** [comment]
I used @Graham-72's PR to generate a new one that's a little closer to the code we were using on http://backdrop-ops.github.io/styleguide so the end result should look more similar.

**Comment by quicksketch** [comment]
I've merged in https://github.com/backdrop/backdrop/pull/843 to move this forward. Let's keep iterating on top of these changes with any additional fixes needed.
